tests/utils/utils.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+describe( 'extend()', function() {
```

Comment:
Do we really need to make the file more bloated, considering that we're already inside the "utils" test page?

Then, the correct name is "utils" (lowercased) and this would make things ackward because there is no dot between "utils" and "extend()": 
"utils extend() should extend and override"

So I think that we should either change the current describe to "utils.extend()" or leave it as it is  now (my option).